docs/getting-started/local-setup/creating-a-keypair.md

```diff
@@ -0,0 +1,75 @@
+---
+id: creating-a-keypair
```

Comment:
this page should demonstrate how to create OR import; in the same page.

Reply:
If you mean import with an existing seed phrase, then it makes sense; if you mean import private key, not sure this is the right place.

Reply:
Will add in another PR 👍